jaseci_ai_kit/jac_speech/jac_speech/vc_tts/README.md

```diff
@@ -1,5 +1,9 @@
 # **Text to Speech with Voice Cloning (`vc_tts`)**
 
+| :warning: WARNING          |
+|:---------------------------|
+| This modules is requires strict dependancies. therefore if you plan to use this with other modules, makesure to install this module in the beggining. Otherwise other modules will break.|
```

Comment:
There are multiple typos and grammatical errors in this sentence. I don't usually pick on these but this one has a little too many.

Reply:
I'll fix